control/veda_utils.py

```diff
@@ -343,3 +343,17 @@ def _FAULT(self, video_object):
 
```

Comment:
Putting this here more for myself to remember, and also for anybody looking at this later:

move_video_within_s3 is just a re-write of the method [move_video](https://github.com/edx/edx-video-pipeline/pull/116/files/afa2f0e0632427713d185e8f96ec898a27565de8#diff-9158dcec44a33a59e532b52e766c6f12L128).

FYI @schenedx